tests/testsuite/alt_registry.rs

```diff
@@ -1362,20 +1367,29 @@ fn both_index_and_registry() {
 #[cargo_test]
 fn both_index_and_default() {
     let p = project().file("src/lib.rs", "").build();
-    for cmd in &[
-        "publish",
-        "owner",
-        "search",
-        "yank --version 1.0.0",
-        "install foo",
-    ] {
+    for cmd in &["publish", "search", "yank --version 1.0.0", "install foo"] {
         p.cargo(cmd)
             .env("CARGO_REGISTRY_DEFAULT", "undefined")
             .arg(format!("--index=index_url"))
             .with_status(101)
             .with_stderr("[ERROR] invalid url `index_url`: relative URL without a base")
             .run();
     }
+    p.cargo("owner --index=index_url add someone")
+        .env("CARGO_REGISTRY_DEFAULT", "undefined")
+        .with_status(101)
+        .with_stderr("[ERROR] invalid url `index_url`: relative URL without a base")
+        .run();
+}
+
+#[cargo_test]
+fn owner_index_and_default() {
+    let p = project().file("src/lib.rs", "").build();
+    p.cargo("owner --index=index_url add someone")
+        .env("CARGO_REGISTRY_DEFAULT", "undefined")
+        .with_status(101)
+        .with_stderr("[ERROR] invalid url `index_url`: relative URL without a base")
+        .run();
 }
```

Comment:
How is this test different than the above test?

Reply:
Changed test cases.